blueflood-core/src/test/java/com/rackspacecloud/blueflood/io/serializers/EnumRollupSerializationTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2015 Rackspace
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+
+
+package com.rackspacecloud.blueflood.io.serializers;
+
+import com.rackspacecloud.blueflood.io.Constants;
+import com.rackspacecloud.blueflood.types.EnumRollup;
+import junit.framework.Assert;
+import org.apache.commons.codec.binary.Base64;
+import org.junit.Test;
+
+import java.io.*;
+import java.nio.ByteBuffer;
+
+public class EnumRollupSerializationTest {
+
+    @Test
+    public void testEnumV1RoundTrip() throws IOException {
+        EnumRollup e0 = new EnumRollup().withObject((long)"enumValue1".hashCode(), (long)1);
+        EnumRollup e1 = new EnumRollup().withObject((long)"t4.enum.abcdefg.hijklmnop.qrstuvw.xyz.ABCDEFG.HIJKLMNOP.QRSTUVW.XYZ.abcdefg.hijklmnop.qrstuvw.xyz.met".hashCode(), 34454722343L);
+
```

Comment:
When testing a container, (like EnumRollups), I prefer to see the test add at least two elements to the container.  Here you create 2 EnumRollups, with one element each.